### DISCUSSION TITLE
Huion GT 221 Pro support?

### DISCUSSION THREAD

**Discussion by mastermaccc** [OpenTabletDriver/OpenTabletDriver#1851] [Q&A] [answered]
I was trying to get my Huion GT 221 Pro working for a game with this driver but sadly discovered that it wasn't supported. Would it be possible for it to get working with this driver?

**Comment by jamesbt365** [comment, marked as answer]
@mastermaccc 
Its possible to do, better done through discord,  though a github issue or here would work as well, If you go to help > export diagnostic and save that file and send it here. Once i have that and a string dump (which i will get after) then i can start. However i will still need you to test and give me more information.
